src/prism.c

```diff
@@ -6263,6 +6263,7 @@ parser_lex_magic_comment_encoding_value(pm_parser_t *parser, const uint8_t *star
                 ENCODING1("macCyrillic", pm_encoding_mac_cyrillic);
                 ENCODING1("macGreek", pm_encoding_mac_greek);
                 ENCODING1("macIceland", pm_encoding_mac_iceland);
+                ENCODING1("macJapanese", pm_encoding_mac_japanese);
```

Comment:
```suggestion
                ENCODING1("MacJapanese", pm_encoding_mac_japanese);
                ENCODING1("MacJapan", pm_encoding_mac_japanese);
```

This one has another alias (`Encoding::MACJAPANESE.names`)

Reply:
No worries!